### PR TITLE
Cleanup playgrounds test driver

### PR DESCRIPTION
**Pull request by adrian-prantl**
@swift-ci test

### CODE REVIEW

**Comment by adrian-prantl** [comment]
I'm seeing mystery errors only in PR testing, I'm hoping that the error assertion will make this more debuggable.

**Comment by adrian-prantl** [comment]
For example: https://github.com/apple/llvm-project/pull/3895

File: lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py [hunk 1]
```diff
@@ -100,9 +79,10 @@ def execute_code(self, inputFile):
             contents = contents_file.read()
 
         result = self.frame.EvaluateExpression(contents, self.options)
-        ouput = self.frame.EvaluateExpression("get_output()")
+        output = self.frame.EvaluateExpression("get_output()")
+        self.assertEquals(str(output.GetError()), 'success')
```
Comment (jimingham) [inline lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: SBError has a Success() method and a .success property.  Either of those seems more straightforward than relying on the str value.

**Review by jimingham — APPROVED** [review]
Except for the trivial cleanup of using SBError.success rather than comparing str output, this looks correct.

File: lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py (hunk 1 above)
Comment (jimingham) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: Note, there's also an lldbtest.assertSuccess() method that asserts the success of an SB Error and forwards the error message to the test output if it failed.  That's probably even more convenient.

Comment (kastiglione) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: > there's also an `lldbtest.assertSuccess()`

TIL!

Comment (adrian-prantl) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: The intention was that if it fails, we would see the stringified error in the comparsion failure.

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py (hunk 1 above)
Comment (adrian-prantl) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: oh! That's even better

Comment (adrian-prantl) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: Due to the extreme latency of the PR testers I might fix that up in a separate PR.

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py (hunk 1 above)
Comment (adrian-prantl) [reply lldb/packages/Python/lldbsuite/test/lldbplaygroundrepl.py]: Done. A random build failrue saved the day!

**Pull request merged**